src/lib/server.ts

```diff
@@ -65,9 +72,23 @@ class Client {
     }
 
     try {
+      if (msg.command === DriverCommand.setApiSchema) {
+        // Handle schema version
+        this.schemaVersion = msg.schemaVersion;
+        if (
+          this.schemaVersion < minSchemaVersion ||
+          this.schemaVersion > maxSchemaVersion
+        ) {
+          throw new SchemaIncompatibleError(this.schemaVersion);
```

Comment:
Instead of throwing we should send an error to the client.

Reply:
Yes, that is handled

Reply:
<img width="1119" alt="Schermafbeelding 2021-02-25 om 20 27 10" src="https://user-images.githubusercontent.com/6389780/109206189-d7e0a680-77a7-11eb-9dab-83befd56b312.png">

Reply:
Ok 👍